src/states/Cutscene.js

```diff
@@ -26,7 +26,6 @@ Cutscene.prototype = {
   preload: function() {
     // Needed for Cutscene
     game.load.script('WorldGenerator', 'src/WorldGenerator.js'); 
-    game.load.spritesheet('loadingImage', 'assets/loadingImage.png', 270, 90, 3);
```

Comment:
Ya it was, if probably just because of an auto merge issue but I do agree. Let's try to be more thorough in tracking changes.